src/components/feedPage/Store.tsx

```diff
@@ -0,0 +1,39 @@
+import S from './Store.style';
+import {StoreType} from '@/types/StoreType';
+import {StackNavigationProp} from '@react-navigation/stack';
+import {RootStackParamList} from '@/types/StackNavigationType';
+import date from '@utils/date';
+
+type Props = {
+  navigation: StackNavigationProp<RootStackParamList, 'Home'>;
+};
+
+const Store = ({navigation, store}: Props & {store: StoreType | null}) => {
+  return (
+    <S.StoreWrapper
+      onPress={() =>
+        navigation.navigate('Detail', {
+          screen: 'Store',
+          params: {storeId: store!.id},
+        })
+      }>
+      <S.StoreImageContainer>
+        {store!.products.map((product, index) => (
+          <S.StoreImageBox key={index}>
+            <S.StoreImage source={{uri: product.image}} />
+            <S.PriceLabel>
+              {product.name}: {product.discountPrice}원
+            </S.PriceLabel>
+          </S.StoreImageBox>
+        ))}
+      </S.StoreImageContainer>
+      <S.StoreTitle>{store!.name}</S.StoreTitle>
+      <S.StorePickupTime>
+        픽업: {date.format(store!.pickupStartAt, 'HH시 mm분')} ~{' '}
+        {date.format(store!.pickupEndAt, 'HH시 mm분')}{' '}
+      </S.StorePickupTime>
```

Comment:
```suggestion
      <S.StorePickupTime>
        {`픽업: ${date.format(store!.pickupStartAt, 'HH시 mm분')} ~ ${date.format(store!.pickupEndAt, 'HH시 mm분')`}
      </S.StorePickupTime>
```

백틱(\`)으로 문자열을 감싸는 Template literals를 사용해서 하면 좋을 것 같습니다. `~{' '}`이러한 불필요한 중괄호, 화이트 스페이스 등을 해결할 수 있겠습니다.